### ISSUE TITLE
Crashes when trying to authenticate while loading a plugin

### ISSUE THREAD

**Issue by joenewstrom** [MscrmTools/XrmToolBox#483]
Version 1.2016.8.23

If authenticated => load plugin, all is well.

If load plugin => "do you want to connect to an organization first?" => yes => enter creds => crash and program exit (reproduced multiple times with multiple plugins)

Windows 10 box high specs, ran toolbox program both with and without elevation.

**Comment by rappen** [comment]
Possibly worthless note from me:
I have seen this on a few occasions too, however not found a pattern. Restart XTB and it usually works this time.

**Comment by airiclenz** [comment]
I confirm this issue and it seems to be still existent in the 2020 version.

**Comment by MscrmTools** [comment]
@airiclenz ok, so the issue would be only if password is not stored, right?

**Comment by airiclenz** [comment]
No, the crash occurs randomly after/while connecting. I will try to run XrmToolBox from Visual Studio to chech if I can replicate the issue and find out what is going wrong.

**Comment by MscrmTools** [comment]
The only point I can see would be connection file saving that fails... but why...
